Doc/library/sqlite3.rst

```diff
@@ -413,6 +413,17 @@ Connection Objects
 
 .. class:: Connection
 
+   Each open SQLite database is represented by a connection object.
+   A connection object is created with :func:`sqlite3.connect`.
```

Comment:
```suggestion
   Each open SQLite database is represented by a ``Connection`` object,
   which is created using :func:`sqlite3.connect`.
```

* Refer to the literal object class
* Avoid repeating ourselves ("connection objects" three times in a row)
* Tweak phrasing to improve flow and concision